packages/typespec-client-generator-core/src/types.ts

```diff
@@ -109,21 +109,15 @@ function addFormatInfo(
       case "uuid":
       case "password":
       case "etag":
+      case "arm-id":
```

Comment:
I thought we agreed that `@format` was not going to be used

Reply:
oh you're right, these are all actual scalar types right?

Reply:
there should be, `password` is just `@secret`, if there is some missing that are needed right now then we should look into adding them as soon as possible.

Reply:
ok made them actual scalar kinds. to clarify, we should never look at the `@format` decorator later down the line right?